### PR TITLE
[ADD] stock_picking_line_sequence v10

### PR DESCRIPTION
**Pull request by serpentcs-dev1**
Stock picking lines with sequence number
========================================

Provide a new field sequence on stock moves, which allows to manage the order of moves in a picking.

### CODE REVIEW

**Comment by serpentcs-dev1** [comment]
@aheficent Could you please review!!

**Comment by serpentcs-dev1** [comment]
@aheficent Still need to add the post_init_hook to update the existing sale orders sequence :)

**Comment by LoisRForgeFlow** [comment]
@darshan-serpent can you please extend the tests a little bit more to reach the target for the codecov?

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
Got this error when duplicating a picking

image

**Comment by serpentcs-dev1** [comment]
@aheficent Tested on runbot, duplicating the record works now!!

Thanks for your functional testing.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Now it's working fine :smile: LGTM. Just please squash the commits before it gets merged.

**Review by LoisRForgeFlow — APPROVED** [review]
Functional and code review :+1:

**Review by espo-tony — APPROVED** [review]

File: stock_picking_line_sequence/models/stock.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    # re-defines the field to change the default
+    sequence = fields.Integer(default=9999)
+
+    # displays sequence on the purchase order line
```
Comment (rousseldenis) [inline stock_picking_line_sequence/models/stock.py]: Could you change the comment?

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: stock_picking_line_sequence/models/stock.py (hunk 1 above)
Comment (serpentcs-dev1) [reply stock_picking_line_sequence/models/stock.py]: @rousseldenis Done, can you please review again?

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by LoisRForgeFlow** [comment]
@darshan-serpent Could you respect the [history of v9 PR](https://github.com/OCA/stock-logistics-workflow/pull/385/commits) and apply your commits here on top? Then we can merge it

**Pull request merged**